shell.nix

```
@@ -0,0 +1,3 @@
+{ pkgs ? import <nixpkgs> { } }:
+
+pkgs.mkShell { nativeBuildInputs = with pkgs; [ pkgconfig gtk3 ]; }
```

```suggestion
pkgs.mkShell { 
  packages = with pkgs; [ pkgconfig gtk3 ]; 
}
```
https://github.com/NixOS/nixpkgs/pull/122180